terraform/rack/gcp/variables.tf

```diff
@@ -55,6 +55,15 @@ variable "syslog" {
   default = ""
 }
 
+variable "telemetry" {
+  default = ""
+}
```

Comment:
you can use this variable as boolean
https://www.devopsschool.com/blog/terraform-variable-boolean-type-explained/